Program.cs

```diff
@@ -40,12 +40,40 @@
                         }
                     };
                 });
+builder.Services.AddAuthorization(options =>
+{
+    options.AddPolicy("IDSCReferenceTokenView", policy =>
```

Comment:
I'm assuming IDSC stands for IdentityServerConfig, but I'm wondering if this prefix is useful. We're already in the scope of the application anyway, so why add the prefix?

Reply:
Your assumption is correct. My thinking was that this way the `<domain>` part of the claim is also included. I've abbreviated IdentityServerConfig due to the length of the name otherwise.

Reply:
I understand, but given that these policies only apply within this particular application anyway I don't think it is necessary to add the IDSC prefix. But perhaps @RVink has a strong opinion on this? ;).

Reply:
fixed in 99a621a